### PR TITLE
Migrating a few stragglers from the TS migration

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Just migrating a few files that were missed as part of the TS migration efforts.

### CODE REVIEW

File: components/activity_log_modal/components/activity_log.test.tsx [hunk 1]
```diff
@@ -32,18 +33,19 @@ describe('components/activity_log_modal/ActivityLog', () => {
     });
 
     test('submitRevoke is called correctly', () => {
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, ActivityLog> = shallow(
```
Comment (hmhealey) [inline components/activity_log_modal/components/activity_log.test.tsx:36]: You can pass a type parameter into `shallow` to simplify the typing
```suggestion
        const wrapper = shallow<ActivityLog>(
```

File: components/activity_log_modal/components/activity_log.test.tsx [hunk 2]
```diff
@@ -32,18 +33,19 @@ describe('components/activity_log_modal/ActivityLog', () => {
...
+        wrapper.instance().submitRevoke(event as unknown as React.MouseEvent);
         expect(baseProps.submitRevoke).toBeCalled();
         expect(baseProps.submitRevoke).toHaveBeenCalledTimes(1);
-        expect(baseProps.submitRevoke).toBeCalledWith('sessionId', 'e');
+        expect(baseProps.submitRevoke).toBeCalledWith('sessionId', event);
     });
 
     test('handleMoreInfo updates state correctly', () => {
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, ActivityLog> = shallow(
```
Comment (hmhealey) [inline components/activity_log_modal/components/activity_log.test.tsx:48]:
```suggestion
        const wrapper = shallow<ActivityLog>(
```

File: components/activity_log_modal/components/activity_log.test.tsx [hunk 3]
```diff
@@ -52,18 +54,18 @@ describe('components/activity_log_modal/ActivityLog', () => {
     });
 
     test('should match when isMobileSession is called', () => {
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, ActivityLog> = shallow(
```
Comment (hmhealey) [inline components/activity_log_modal/components/activity_log.test.tsx:57]:
```suggestion
        const wrapper = shallow<ActivityLog>(
```

File: components/activity_log_modal/components/activity_log.test.tsx [hunk 4]
```diff
@@ -52,18 +54,18 @@ describe('components/activity_log_modal/ActivityLog', () => {
...
-        expect(isMobileSession({device_id: 'android'})).toEqual(true);
-        expect(isMobileSession({device_id: 'none'})).toEqual(false);
+        expect(isMobileSession(TestHelper.getSessionMock({device_id: 'apple'}))).toEqual(true);
+        expect(isMobileSession(TestHelper.getSessionMock({device_id: 'android'}))).toEqual(true);
+        expect(isMobileSession(TestHelper.getSessionMock({device_id: 'none'}))).toEqual(false);
     });
 
     test('should match when mobileSessionInfo is called', () => {
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, ActivityLog> = shallow(
```
Comment (hmhealey) [inline components/activity_log_modal/components/activity_log.test.tsx:68]:
```suggestion
        const wrapper = shallow<ActivityLog>(
```

File: components/activity_log_modal/components/activity_log.test.tsx [hunk 5]
```diff
@@ -2,24 +2,25 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallow} from 'enzyme';
+import {shallow, ShallowWrapper} from 'enzyme';
```
Comment (hmhealey) [inline components/activity_log_modal/components/activity_log.test.tsx:5]:
```suggestion
import {shallow} from 'enzyme';
```

File: components/common/infinite_scroll.test.tsx [hunk 6]
```diff
@@ -17,10 +18,10 @@ describe('/components/common/InfiniteScroll', () => {
         pageNumber: 1,
     };
 
-    let wrapper;
+    let wrapper: ReactWrapper<any, any, InfiniteScroll>;
 
     beforeEach(() => {
-        wrapper = mountWithIntl(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>);
+        wrapper = mountWithIntl(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>) as unknown as ReactWrapper<any, any, InfiniteScroll>;
```
Comment (hmhealey) [inline components/common/infinite_scroll.test.tsx:24]: I think mountWithIntl should also support that type parameter as well. I'm not 100% sure though
```suggestion
        wrapper = mountWithIntl<InfiniteScroll>(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>);
```

File: components/create_team/components/display_name.test.tsx [hunk 7]
```diff
@@ -60,7 +60,7 @@ describe('/components/create_team/components/display_name', () => {
             },
         };
 
-        wrapper.find('.form-control').instance().value = teamDisplayName;
+        (wrapper.find('.form-control') as unknown as ReactWrapper<any, any, HTMLInputElement>).instance().value = teamDisplayName;
```
Comment (hmhealey) [inline components/create_team/components/display_name.test.tsx:63]: Now I'm just making stuff up that might work
```suggestion
        wrapper.find<HTMLInputElement>('.form-control').instance().value = teamDisplayName;
```

File: components/create_team/components/display_name.test.tsx [hunk 8]
```diff
@@ -73,7 +73,7 @@ describe('/components/create_team/components/display_name', () => {
 
     test('should display isRequired error', () => {
         const wrapper = mountWithIntl(<DisplayName {...defaultProps}/>);
-        wrapper.find('.form-control').instance().value = '';
+        (wrapper.find('.form-control') as unknown as ReactWrapper<any, any, HTMLInputElement>).instance().value = '';
```
Comment (hmhealey) [inline components/create_team/components/display_name.test.tsx:76]:
```suggestion
        wrapper.find<HTMLInputElement>('.form-control').instance().value = '';
```

File: components/create_team/components/display_name.test.tsx [hunk 9]
```diff
@@ -101,7 +101,7 @@ describe('/components/create_team/components/display_name', () => {
 
         const wrapper = mountWithIntl(<DisplayName {...nullTeamProps}/>);
 
-        wrapper.find('.form-control').instance().value = '';
+        (wrapper.find('.form-control') as unknown as ReactWrapper<any, any, HTMLInputElement>).instance().value = '';
```
Comment (hmhealey) [inline components/create_team/components/display_name.test.tsx:104]:
```suggestion
        wrapper.find<HTMLInputElement>('.form-control').instance().value = '';
```

File: components/create_team/components/display_name.test.tsx [hunk 10]
```diff
@@ -130,7 +130,7 @@ describe('/components/create_team/components/display_name', () => {
...
```
Comment (hmhealey) [inline components/create_team/components/display_name.test.tsx:133]:
```suggestion
        wrapper.find<HTMLInputElement>('.form-control').instance().value = '';
```

File: components/create_team/components/display_name.test.tsx [hunk 11]
```diff
@@ -147,7 +147,7 @@ describe('/components/create_team/components/display_name', () => {
 
     test('should display charLength error', () => {
         const wrapper = mountWithIntl(<DisplayName {...defaultProps}/>);
-        const input = wrapper.find('.form-control').instance();
+        const input = (wrapper.find('.form-control') as unknown as ReactWrapper<any, any, HTMLInputElement>).instance();
```
Comment (hmhealey) [inline components/create_team/components/display_name.test.tsx:150]:
```suggestion
        const input = wrapper.find<HTMLInputElement>('.form-control').instance();
```

File: components/favicon_title_handler/favicon_title_handler.test.tsx [hunk 12]
```diff
@@ -26,26 +31,26 @@ describe('components/FaviconTitleHandler', () => {
...
-            type: 'team-type',
-        },
+            type: 'team-type' as TeamType,
+        }),
         currentTeammate: null,
     };
 
     test('set correctly the title when needed', () => {
-        const wrapper = shallowWithIntl(
+        const wrapper: ShallowWrapper<any, any, FaviconTitleHandlerClass> = shallowWithIntl(
```
Comment (hmhealey) [inline components/favicon_title_handler/favicon_title_handler.test.tsx:51]: GitHub won't let me do suggestions on these ones

**Review by hmhealey — CHANGES_REQUESTED** [review]

File: components/create_team/components/display_name.test.tsx (hunk 7 above)
Comment (devinbinnie) [reply components/create_team/components/display_name.test.tsx:63]: Nope lol

File: components/create_team/components/display_name.test.tsx (hunk 8 above)
Comment (devinbinnie) [reply components/create_team/components/display_name.test.tsx:76]: Nope

File: components/favicon_title_handler/favicon_title_handler.test.tsx (hunk 12 above)
Comment (devinbinnie) [reply components/favicon_title_handler/favicon_title_handler.test.tsx:51]: Also doesn't work

File: components/common/infinite_scroll.test.tsx (hunk 6 above)
Comment (devinbinnie) [reply components/common/infinite_scroll.test.tsx:24]: Nope

**Review by hmhealey — APPROVED** [review]

**Review by calebroseland — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Thanks @devinbinnie 
Tested, looks good to merge.
- Verified [E2E run](https://mm-cypress-report.s3.amazonaws.com/4405-webapp-pr-7770-mattermost-enterprise-edition:31a64f8/mochawesome.html) results
- Manually re-tested one case that failed, differs from E2E master branch results, [MM-T946](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T946),  where user should continue to be listed on LHS after deactivation - working as expected on the test server.

**Pull request merged**